pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/scheduler/TransferShedder.java

```diff
@@ -434,4 +466,37 @@ private boolean isTransferable(String bundle) {
         }
         return true;
     }
+
+    /**
+     * Check the gave bundle and broker can be transfer or unload with isolation policies applied.
+     *
+     * @param context The load manager context.
+     * @param availableBrokers The available brokers.
+     * @param namespaceBundle The bundle try to unload or transfer.
+     * @param maxBroker The current broker.
+     * @param minBroker The broker will be transfer to.
+     * @return Can be transfer/unload or not.
+     */
+    private boolean canTransferWithIsolationPoliciesToBroker(LoadManagerContext context,
+                                                             Map<String, BrokerLookupData> availableBrokers,
+                                                             NamespaceBundle namespaceBundle,
+                                                             String maxBroker,
+                                                             Optional<String> minBroker) {
+        if (isolationPoliciesHelper == null
```

Comment:
’maxBroker‘ -> 'currentBroker' 
'minBroker' -> 'targetBroker', would be better ?

Reply:
Ah, many place use the same name

Reply:
Updated.